manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
+PackageVersion: 3.3.499.0
+PackageLocale: en-US
+Publisher: SecureMix LLC
+PackageName: GlassWire
+License: Copyright SecureMix LLC

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate